### ISSUE TITLE
Simple usage example

### ISSUE THREAD

**Issue by dmitriy-serdyuk** [dmlc/tensorboard#14]
Hi, thanks for a very nice work!

After looking for some time at your project I realized that it is completely independent from either Tensorflow or Mxnet, one can use it with Theano (like in my case), Pytorch or even standalone python scripts.

I propose to add an example which demonstrates how to use it in the simplest way, something like:
```python
import tensorboard
from tensorboard import summary
train_writer = tensorboard.FileWriter('./log/train')
test_writer = tensorboard.FileWriter('./log/test')
for i in range(10):
    train_writer.add_summary(summary.scalar('accuaracy', i ** 2), global_step=i)
    test_writer.add_summary(summary.scalar('accuaracy', (i ** 2) * .9), global_step=i)
```

Let me know what you think, I can create a PR.

**Comment by zihaolucky** [comment]
Hi @dmitriy-serdyuk 

Yes, it's a standalone version of TensorBoard and could be used in general visualization purpose.

You're welcomed to contribute more examples/tutorials, we put the materials under `tensorboard/docs/tutorial/`. I've written a tutorial *understanding-vanish-gradient* [here](https://github.com/dmlc/tensorboard/blob/master/docs/tutorial/understanding-vanish-gradient.ipynb) Or you could update the [README](https://github.com/dmlc/tensorboard/tree/master/python).

**Comment by zihaolucky** [comment]
Just re-open if you need, close for now.
